Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -405,6 +405,57 @@ public override void HandleEvent<T>(BaseEventData eventData, ExecuteEvents.Event
             }
         }
 
+        /// <summary>
+        /// Handles a pre and post focus events
+        /// We send all focus events to all global listeners and the actual focus change receivers. the use flag is completely ignored to avoid any interception.
+        /// </summary>
+        public void HandleEvent<T>(FocusEventData focusEventData, ExecuteEvents.EventFunction<T> eventHandler) where T : IEventSystemHandler
```

Comment:
Can this be made private? Maybe also name it HandleFocusChangedEvents for added clarity.